src/app/profile.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+import data from './mock-profile.json';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ProfileService {
+
+  constructor() { }
+
+  getProfile(): Observable<any> {
```

Comment:
in general, I would recommend creating a typescript interface type when possible.  Then you don't have to return type "any".  So if you create an interface called "Profile" that matches your mock, you can return Observable<Profile>.  Makes the development experience much more effective and tooling in vscode is greatly enhanced.

Reply:
I saw later you tried that approach, saw this question first.  :)